### PR TITLE
docs: add Pomerium

### PR DESCRIPTION
**Pull request by desimone**
RE: https://github.com/pomerium/pomerium/pull/2223

## Checklist

- [ ] updated `CHANGELOG.md`

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ code-server only supports password authentication natively.
 **note**: code-server will rate limit password authentication attempts at 2 a minute and 12 an hour.
 
 If you want to use external authentication (i.e sign in with Google) you should handle this
-with a reverse proxy using something like [oauth2_proxy](https://github.com/pusher/oauth2_proxy)
+with a reverse proxy using something like [Pomerium](https://www.pomerium.io/guides/vs-code-server.html), [oauth2_proxy](https://github.com/pusher/oauth2_proxy)
```
Comment (jsjoeio) [inline docs/FAQ.md:186]: Will the URL need to change once https://github.com/pomerium/pomerium/pull/2223 is merged?
```suggestion
with a reverse proxy using something like [Pomerium](https://www.pomerium.io/guides/code-server.html), [oauth2_proxy](https://github.com/pusher/oauth2_proxy)
```

**Review by jsjoeio — CHANGES_REQUESTED** [review]
Thanks for adding this! 🙌

One minor issue with URL (I think)

File: docs/FAQ.md (hunk 1 above)
Comment (desimone) [reply docs/FAQ.md:186]: Actually -- I think the name of the file would have to change as well. And we should add a redirect so it doesn't break anyone's bookmarks. 

- https://github.com/pomerium/pomerium/blob/master/docs/.vuepress/config.js#L170
- https://github.com/pomerium/pomerium/blob/master/docs/.vuepress/public/_redirects#L33

or we could keep the original uri and just add a redirect. LMK

Comment (jsjoeio) [reply docs/FAQ.md:186]: Oh! Good point. Okay, I'll make those changes in my PR

Comment (jsjoeio) [reply docs/FAQ.md:186]: Made the changes! Thanks for the guidance!

**Review by jsjoeio — APPROVED** [review]
LGTM! Thanks so much @desimone 👍🏼

**Comment by jsjoeio** [comment]
If you want to update the `CHANGELOG` with your change under "New Version" > "Documentation", then we'll make sure to credit you in the Release Notes for this update. Not required though :)

**Pull request merged**